spec/db/models.rb

```diff
@@ -26,7 +26,13 @@ class Author < VirtualTotalTestBase
            :class_name => "Book", :foreign_key => "author_id"
 
   virtual_total :total_recently_published_books, :recently_published_books
+  virtual_average :average_recently_published_books, :recently_published_books, :rating
+  virtual_minimum :maximum_recently_published_books, :recently_published_books, :rating
+  virtual_maximum :minimum_recently_published_books, :recently_published_books, :rating
   virtual_aggregate :sum_recently_published_books_rating, :recently_published_books, :sum, :rating
```

Comment:
maybe introduce `virtual_sum` (or pick a better name)

lets move away from the aggregate concept. At least for the public.

Rails has a calculate method. Users use `maximum` and `minimum` rather than `calculate(:maximum, ...)`
I think we should do the same